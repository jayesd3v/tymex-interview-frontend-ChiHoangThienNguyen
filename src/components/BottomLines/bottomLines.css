.bottom-lines {
    background: url(/public/bottom-lines.png) no-repeat;
    background-position: bottom center;
    background-size: contain;
    height: 300px;
}
