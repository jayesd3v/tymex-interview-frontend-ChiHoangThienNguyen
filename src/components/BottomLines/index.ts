import BottomLines from './BottomLlines';

export default BottomLines;
