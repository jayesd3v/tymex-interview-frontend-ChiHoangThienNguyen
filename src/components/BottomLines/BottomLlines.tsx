import './bottomLines.css';

const BottomLines = () => {
    return <div className='bottom-lines'></div>;
};

export default BottomLines;
