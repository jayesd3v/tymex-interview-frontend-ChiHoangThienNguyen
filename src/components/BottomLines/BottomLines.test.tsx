import { render } from "@testing-library/react";
import BottomLines from "./BottomLlines";


describe('BottomLines tests', () => {
    test('snapshot test', () => {
        const { container } = render(<BottomLines />);
        expect(container).toMatchSnapshot();
    });
});