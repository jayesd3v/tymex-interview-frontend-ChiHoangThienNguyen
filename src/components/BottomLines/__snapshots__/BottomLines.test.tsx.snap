// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomLines tests snapshot test 1`] = `
<div>
  <div
    class="bottom-lines mt-5"
  />
</div>
`;
